File: 1011 - volume_esfera/esfera.scala
object Main{
    def vesfera( raio: Double) : Double =
        (4.0 / 3.0 ) * 3.14159 * raio * raio * raio
    
    def main( args: Array[String]){
        var raio = scala.io.StdIn.readDouble : Double
        var vol = vesfera(raio)
        printf("VOLUME = %.3f\n", vol)
    }
}

File: 1011 - volume_esfera/esfera.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

function vesfera ( raio ){
    return ((4.0/3.0) * 3.14159 * raio * raio * raio );
}

var raio = parseInt( lines );
var vol = vesfera(raio);
console.log("VOLUME = " + vol.toFixed(3));
